bundles/org.openhab.binding.freeboxos/src/main/java/org/openhab/binding/freeboxos/internal/handler/ActivePlayerHandler.java

```diff
@@ -73,12 +73,14 @@ protected void internalPoll() throws FreeboxException {
             Player player = getManager(PlayerManager.class).getDevice(getClientId());
             updateStatus(player.reachable() ? ThingStatus.ONLINE : ThingStatus.OFFLINE);
             if (player.reachable()) {
-                Status status = getManager(PlayerManager.class).getPlayerStatus(getClientId());
-                if (status != null) {
-                    updateChannelString(PLAYER_STATUS, PLAYER_STATUS, status.powerState().name());
-                    ForegroundApp foreground = status.foregroundApp();
-                    if (foreground != null) {
-                        updateChannelString(PLAYER_STATUS, PACKAGE, foreground._package());
+                if (isLinked(PLAYER_STATUS + "#" + PLAYER_STATUS) || isLinked(PLAYER_STATUS + "#" + PACKAGE)) {
```

Comment:
Here and other places: `ChannelUID.CHANNEL_GROUP_SEPARATOR` could be used rather than hardcoded `#`.

Reply:
Done